openapi/src/parts/controlled_azure_database.yaml

```diff
@@ -88,9 +88,8 @@ components:
           type: string
           pattern: ^[a-zA-Z][a-zA-Z0-9_]{0,31}$
         owner:
-          description: Resource id of an azure managed identity in this workspace to be the owner of the database, required for shared databases. For private databases, the owner will be determined from privateResourceUser.
+          description: Resource id or name of an azure managed identity in this workspace to be the owner of the database, required for shared databases. For private databases, the owner will be determined from privateResourceUser.
```

Comment:
How can the system distinguish names from IDs? Is it possible to get a collision between a name and an ID on different databases?

Reply:
if it is a uuid and exists in the workspace, then it is treated as an id, otherwise a name